flight_phase_files/HALO_RF04_20200126_info.yaml

```
@@ -21,71 +26,366 @@ remarks:
+    - HALO-0126_s21
+    - HALO-0126_s22
+    - HALO-0126_s23
+    - HALO-0126_s24
+    - HALO-0126_s25
+    - HALO-0126_s26
+    - HALO-0126_s27
+    - HALO-0126_s28
+    - HALO-0126_s29
+    - HALO-0126_s30
+    - HALO-0126_s31
+    - HALO-0126_s32
+    - HALO-0126_s33
+    - HALO-0126_s34
+    - HALO-0126_s36
+    BAD:
+    - HALO-0126_s19
+    UGLY:
+    - HALO-0126_s13
+    - HALO-0126_s35
+- kinds:
+  - straight_leg
+  name: Leg-1 over Meteor
+  irregularities:
+  - slightly high roll towards end of the leg
+  segment_id: HALO-0126sl1
   start: 2020-01-26 15:50:46
   end: 2020-01-26 16:04:52
-- kind: straight_leg
-  name: leg2_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY:
+    - HALO-0126_s37
+- kinds:
+  - straight_leg
+  name: Leg-2 over Meteor
+  irregularities:
+  - slightly high roll at start of leg
+  segment_id: HALO-0126sl2
   start: 2020-01-26 16:11:14
   end: 2020-01-26 16:31:14
-- kind: straight_leg
-  name: leg3_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - straight_leg
+  name: Leg-3 over Meteor
+  irregularities: []
+  segment_id: HALO-0126sl3
   start: 2020-01-26 16:37:21
   end: 2020-01-26 16:52:17
-- kind: straight_leg
-  name: leg4_over_Meteor
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - straight_leg
+  name: Leg-4 over Meteor
+  irregularities: []
+  segment_id: HALO-0126sl4
   start: 2020-01-26 16:58:46
   end: 2020-01-26 17:14:14
-- kind: radar_calibration_wiggle
-  name: radar_cal_wiggle
+  dropsondes:
+    GOOD:
+    - HALO-0126_s38
+    - HALO-0126_s39
+    BAD: []
+    UGLY: []
+- kinds:
+  - radar_calibration_wiggle
+  name: Radar Calibration Wiggle
+  irregularities: []
+  segment_id: HALO-0126rcw1
   start: 2020-01-26 17:14:14
-  end: 2020-01-26 17:21:01
-- kind: circle
-  name: c4
-  start: 2020-01-26 17:21:01
-  end: 2020-01-26 18:12:50
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-26 18:12:50
-  end: 2020-01-26 18:36:01
-- kind: circle
-  name: c5
-  start: 2020-01-26 18:36:01
-  end: 2020-01-26 19:29:21
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-26 19:29:21
-  end: 2020-01-26 19:41:01
-- kind: circle
-  name: c6
-  start: 2020-01-26 19:51:32
-  end: 2020-01-26 20:45:42
-- kind: lidar_calibration
-  name: lidar_cal
+  end: 2020-01-26 17:16:15
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: Circle-4
+  irregularities: []
+  segment_id: HALO-0126c4
+  start: 2020-01-26 17:19:54
+  end: 2020-01-26 18:18:34
+  dropsondes:
+    GOOD:
+    - HALO-0126_s40
+    - HALO-0126_s41
+    - HALO-0126_s42
+    - HALO-0126_s43
+    - HALO-0126_s44
+    - HALO-0126_s45
+    - HALO-0126_s46
+    - HALO-0126_s47
+    - HALO-0126_s48
+    - HALO-0126_s49
+    - HALO-0126_s50
+    - HALO-0126_s51
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle_break
+  name: Circle break between circle-4 and circle-5
+  irregularities: []
+  segment_id: HALO-0126cb3
+  start: 2020-01-26 18:18:35
```

If the end of c4 is changed, then the start of cb3 should be changed accordingly. 
```suggestion
  start: 2020-01-26 18:17:54
```